### ISSUE TITLE
Proposal to add PageView.separated

### ISSUE THREAD

**Issue by tbm98** [flutter/flutter#79018]
I want to use PageView with a separate view like ListView.separated

**Comment by darshankawar** [comment]
@tbm98 
Can you elaborate on your use case that need `PageView` to have such feature ?
Thanks.

**Comment by Iampato** [comment]
I have one scenario a pageview with a vertical Scroll direction and has a viewport of 0.8 adding a separator widget would be nice

**Comment by tbm98** [comment]
> I have one scenario a pageview with a vertical Scroll direction and has a viewport of 0.8 adding a separator widget would be nice

@Iampato @darshankawar yes, not just in vertical but when `viewport` < 1.0 I want to display something between each page :D 
ex: the fixed space between the banners

**Comment by maheshmnj** [comment]
Assigning this issue to myself will send a PR tonight
